doc/cli/npm-help.md

```diff
@@ -5,6 +5,8 @@ npm-help(1) -- Get help on npm
 
     npm help <term> [<terms..>]
 
+    aliases: apihelp
```

Comment:
we should probably nuke `apihelp` as an alias / leave this undocumented, as there currently is no meaningful API documentation (on purpose).